Sources/TuistGenerator/Generator/BuildPhaseGenerator.swift

```diff
@@ -514,20 +514,21 @@ final class BuildPhaseGenerator: BuildPhaseGenerating {
         fileElements: ProjectFileElements,
         pbxproj: PBXProj
     ) throws {
-        let appExtensions = graphTraverser.appExtensionDependencies(path: path, name: target.name).sorted()
+        let appExtensions = graphTraverser.appExtensionDependenciesWithConditions(path: path, name: target.name)//.sorted()
```

Comment:
we need sorting to take place somewhere in the pipeline to ensure stable project generation 🤐

Reply:
Added sorting again